routes/web.php

```diff
@@ -48,7 +48,7 @@
 Route::get('servicio/activo/{id}', 'ServicioController@finalizarActivo')->name('servicio.finalizarActivo');
 route::get('servicio/finalizado', 'ServicioController@finalizado')->name('servicio.finalizado');
 Route::get('servicio/estadistica', 'ServicioController@estadistica')->name('servicio.estadistica');
-Route::get('servicio/{mes}/{anio}/tabla', 'ServicioController@tabla')->name('servicio.tabla');
+Route::get('servicio/{monthSince}/{yearSince}/{monthUntil}/{yearUntil}/tabla', 'ServicioController@tabla')->name('servicio.tabla');
```

Comment:
Lo ideal es que estos parametros, en lugar de pasarlos `/monthSince/yearSince...`, deberiamos pasarlos como parametros GET `servicio/tabla/?monthSince=12&yearSince=2008....` ya que son filtros al endpoint de tabla. Algo como esto: https://laracasts.com/discuss/channels/general-discussion/how-to-create-a-laravel-restful-api-allowing-requests-with-url-params?page=1.  Estaria bueno poder hacerlo, pero no es obligación.

Reply:
Ahí miro esto y lo intento hacer. No lo voy a poder probar porque estoy de viaje y la maquina a la que tengo acceso no tengo linux para levantar los containers y ejecutar la app. Calculo que si pasa Travis quiere decir que quedo bien no?

Reply:
Nacho, una consulta, los valores esos que me decis, de donde los saco o como los paso? seria traidos del imput pero osea no puedo pasarle el valor tendria que pasarle el nombre de la variable como lo estoy haciendo, pero solo modificar la forma de llamado a la ruta? que tambien deberia modificarlo en ajaxtabla.js  

lo que quiero decir es : servicio/tabla/?monthSince=VARIABLE&yearSince=VARIBLE.... algo asi seria?

Reply:
Mas vale dejalo así, creo un ticket aparte para cambiarlo, porque no tenes la maquina para probalo.